dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -567,11 +612,14 @@ protected void loadPermissions(final Contentlet con, final Map<String,Object> m)
 		}
 		m.put(ESMappingConstants.PERMISSIONS, permissionsSt.toString());
 		m.put(ESMappingConstants.OWNER_CAN_READ, ownerCanRead);
-		m.put(ESMappingConstants.OWNER_CAN_READ + TEXT, Boolean.toString(ownerCanRead));
 		m.put(ESMappingConstants.OWNER_CAN_WRITE, ownerCanWrite);
-		m.put(ESMappingConstants.OWNER_CAN_WRITE + TEXT, Boolean.toString(ownerCanWrite));
 		m.put(ESMappingConstants.OWNER_CAN_PUBLISH, ownerCanPub);
-		m.put(ESMappingConstants.OWNER_CAN_PUBLISH + TEXT, Boolean.toString(ownerCanPub));
+
+        if (Config.getBooleanProperty("CREATE_TEXT_INDEX_FIELD_FOR_NON_TEXT_FIELDS", false)) {
+            m.put(ESMappingConstants.OWNER_CAN_READ + TEXT, Boolean.toString(ownerCanRead));
```

Comment:
m is missingless, rename it

Reply:
@jdotcms what do you mean with "m is missingless"?

Reply:
I think he meant "meaningless" and that you should perhaps rename the variable "map"  or "permissionsMap" or something

Reply:
Done